doc/manual/manual.tex

```diff
@@ -565,9 +566,15 @@ \subsubsection{Boundary conditions}
 $\Gamma_{0,\mathbf u}\cup\Gamma_{\text{prescribed},\mathbf
   u}\cup\Gamma_{\parallel,\mathbf u}\cup\Gamma_{\text{traction},\mathbf
   u}=\Gamma$ and
-$\Gamma_{D,T}\cup\Gamma_{N,T}=\Gamma$. No boundary conditions have to be
-posed for the compositional fields at those parts of the boundary where flow is either
-tangential to the boundary or points outward.
+$\Gamma_{D,T}\cup\Gamma_{N,T}=\Gamma$. 
+
+Boundary conditions have to be posed for the compositional fields only
```

Comment:
pose/impose, same thing I guess in this context. But I'm happy to say "impose".

I'm scratching my head about the inflow/outflow thingy. It looks like we're imposing boundary conditions on all Dirichlet components, whether in- or outflow. Since the text is now mathematically correct, I'll open another issue to investigate this further.